lib/bitex/client.rb

```diff
@@ -0,0 +1,123 @@
+module Bitex
+  # This client connects via API to Bitex resources.
+  class Client
+    attr_reader :api_key
+
+    def initialize(api_key: nil, sandbox: false)
+      @api_key = api_key
+      environment(sandbox)
+    end
+
+    def environment(sandbox)
```

Comment:
NITPICK: this method has side-effects, let's rename it to something that makes it clear it will change something like `setup_environment` or similar.